pkgs/tools/misc/symbolator/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, python27Packages }:
+
+python27Packages.buildPythonApplication rec {
```

Comment:
Is it me or they do support Python 3? https://github.com/kevinpt/symbolator/blob/a9b24e4d6cf2a5369cd2a4a9604025f1c9e2ecdb/setup.py#L58

Reply:
Yes, but symbolator requires pango which is available in either PyGTK, which appears to only have Python 2 bindings, or PyGObject which has Python 3 bindings but created runtime exceptions.

Reply:
Symbolator lies in this awkward zone of trying to support both Python 2 and Python 3.
Would it be wise to fork symbolator, update it to use Python 3, and try to build it with Py3 packages?

Reply:
If you can fix the program yourself, to make it fully functional with Python 3 and PyGObject, even if it'd require using an "unstable" release, that'd be awesome. I don't think Nixpkgs is upto adding packages with such deprecated libraries, see https://github.com/NixOS/nixpkgs/issues/75478. 

If you do so, please contribute your work upstream, and when that's done, please don't switch to fetching `src` from your fork directly, but use `patches` to include your suggestions, with a link to the upstream PR.